test-helpers/src/connection/cassandra/cql_ws.rs

```diff
@@ -228,25 +231,28 @@ fn spawn_read_write_tasks<S: AsyncRead + AsyncWrite + Unpin + Send + 'static>(
     });
 }
 
-fn load_ca(path: &str) -> RootCertStore {
-    let mut pem = BufReader::new(File::open(path).unwrap());
-    let certs = rustls_pemfile::certs(&mut pem).unwrap();
-
+fn load_ca(path: &str) -> Result<RootCertStore> {
+    let mut pem = BufReader::new(File::open(path)?);
     let mut root_cert_store = RootCertStore::empty();
-    for cert in certs {
-        root_cert_store.add(&Certificate(cert)).unwrap();
+    for cert in rustls_pemfile::certs(&mut pem) {
+        root_cert_store
+            .add(cert.context("Error while parsing PEM")?)
+            .context("Failed to add cert to cert store")?;
     }
-    root_cert_store
+    Ok(root_cert_store)
 }
 
+#[derive(Debug)]
 pub struct SkipVerifyHostName {
-    verifier: WebPkiVerifier,
+    verifier: Arc<WebPkiServerVerifier>,
```

Comment:
Why do we need Arc here?

Reply:
Its because `WebPkiServerVerifier::builder(Arc::new(roots)).build()` returns an Arc which is part of rustls API, and we need to store the value it returns here.